src/praisonai-code/praisonai_code/cli/commands/run.py

```
@@ -1076,6 +1076,25 @@ def run_main(
+        try:
+            from ..state.project_sessions import find_last_session, find_session_model
+
+            resumed_id = session or find_last_session()
+            if resumed_id:
+                recorded = find_session_model(resumed_id)
+                if recorded:
+                    model = recorded
+                    output.print_info(f"Restored session model: {model}")
+        except Exception:
+            # Model restore is best-effort; fall back to default resolution.
+            pass
```

Ruff (0.16.1)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)